### PR TITLE
MYB-170: Populates FavoriteCardView with the DAO's items

### PR DESCRIPTION
**Pull request by jcg22cabeza**

### CODE REVIEW

**Comment by quiquep** [comment]
Looks good to me.

File: app/src/main/java/com/mybus/view/FavoritesCardView.java [hunk 1]
```diff
@@ -31,28 +37,33 @@ public FavoritesCardView(Context context, AttributeSet attrs) {
...
 
-        // specify an adapter (see also next example)
-        String[] myDataSet = {"Casa", "Trabajo", "Gym", "Casa Novia", "Casa", "Trabajo", "Gym", "Casa Novia"};
-        mListAdapter = new FavoriteItemAdapter(myDataSet);
-        mListAdapter.setItemSelectedListener(this);
-        mFavoriteItemsList.setAdapter(mListAdapter);
+            mListAdapter = new FavoriteItemAdapter(mFavoriteLocationList);
+            mListAdapter.setItemSelectedListener(this);
+            mRecyclerView.setAdapter(mListAdapter);
+        }
```
Comment (ldimitroff) [inline app/src/main/java/com/mybus/view/FavoritesCardView.java:63]: Should have the:
`else{
         mNoFavoritesTextView.setVisibility(VISIBLE);
         mRecyclerView.setVisibility(GONE);
}`

File: app/src/main/java/com/mybus/view/HistoryCardView.java [hunk 2]
```diff
@@ -53,21 +53,18 @@ public void setList(List<RecentLocation> list) {
...
             // use a linear layout manager
             mLayoutManager = new LinearLayoutManager(getContext());
-            mHistoryItemsList.setLayoutManager(mLayoutManager);
+            mRecyclerView.setLayoutManager(mLayoutManager);
 
             mListAdapter = new HistoryItemAdapter(mRecentLocationList);
             mListAdapter.setItemSelectedListener(this);
-            mHistoryItemsList.setAdapter(mListAdapter);
-        } else {
-            mHistoryItemsList.setVisibility(GONE);
```
Comment (ldimitroff) [inline app/src/main/java/com/mybus/view/HistoryCardView.java:69]: Same as above, if you're setting the view a visibility, yo need to change it in the else, no matter on what state you setted it by the XML

File: app/src/main/java/com/mybus/view/HistoryCardView.java [hunk 3]
```diff
@@ -53,21 +53,18 @@ public void setList(List<RecentLocation> list) {
...
-            mNoRecentTextView.setVisibility(VISIBLE);
+            mRecyclerView.setAdapter(mListAdapter);
```
Comment (ldimitroff) [inline app/src/main/java/com/mybus/view/HistoryCardView.java]: at this point do you know mListAdapter value?

**Pull request merged**